packages/wallet-ui/src/utils/utils.ts

```diff
@@ -87,20 +83,15 @@ export const getHumanReadableAmount = (
 };
 
 export const getSpendableTotalBalance = (asset: Erc20TokenBalance): string => {
```

Comment:
nit: i guess this method can be remove and rollback to getHumanReadableAmount from the caller? but im also okay to keep this here